.gitignore

```
@@ -35,6 +35,8 @@ yarn-error.log*
 .DS_Store
 *.pem
 
+*storybook.log
 # yarn
 .yarn/*
-apps/root-admin/.yarn/*
\ No newline at end of file
+apps/root-admin/.yarn/*
```

```suggestion
apps/root-admin/.yarn/*
!.yarn/cache // Zero-Install 
!.yarn/patches
!.yarn/plugins
!.yarn/releases
!.yarn/sdks
!.yarn/versions
```
https://velog.io/@sangbooom/%ED%94%84%EB%A1%9C%EC%A0%9D%ED%8A%B8-yarn-berry%EB%A1%9C-%EB%A7%88%EC%9D%B4%EA%B7%B8%EB%A0%88%EC%9D%B4%EC%85%98-%ED%95%98%EA%B8%B0#yarn-install-%EC%8B%9C-cannot-find-module--%EC%97%90%EB%9F%AC

해당 블로그에서 참고했는데, 해당 설정도 추가하면 좋을 듯 합니다.!

---

고생하셨습니다. 리뷰 양이 많으니 리뷰 작성에도 렉이 심해서 어렵네요! ㅜㅜㅜ